Documentation/auto-skip.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Auto-Skip
+---
+
+![header_getting_started](/media/headers/autoadvance.png)
+
+## What is Auto-Skip?
+Auto-Skip is the concept of advancing a timeline until an unseen text or
```

Comment:
I'd say at it's core it's the concept of advancing a timeline quicker then you can read. Like fast-forwarding. Using it to get to unread sections is just a likely usecase, not the only way it can be used.